crates/python_extractor/src/main.rs

```diff
@@ -123,12 +123,16 @@ async fn main() -> Result<()> {
 }
 
 fn expand_path_to_defs_from_offset(from_given_path: &str, path: &str) -> Vec<String> {
-    if let Some(rem) = path.strip_prefix(from_given_path) {
+    if let Some(rem0) = path.strip_prefix(from_given_path) {
+        let rem = match rem0.strip_prefix("site-packages/") {
```

Comment:
```
if let Some(rem) = path.strip_prefix(from_given_path).and_then(|p| p.strip_prefix("site-packages/").unwrap_or(p) {
```
inner block is ok, too but i'd use `unwrap_or` here. Probably also just put your comment here into the file

Reply:
Addressed in 456bc96